python/paddle/fluid/contrib/mixed_precision/fp16_utils.py

```diff
@@ -209,19 +209,23 @@ def find_true_prev_op(ops, cur_op, var_name):
     return None
 
 
-def find_true_post_op(ops, cur_op, var_name):
+def find_true_post_op(ops, cur_op, var_name, search_all=False):
     """
     if there are post ops, return them, if there is no post op,
     return None instead.
     Args:
         ops (list): A list of ops.
         cur_op (Operator): Current operator which has var_name variable.
         var_name (string): Variable name.
+        search_all (bool): The type of operator search. Use if \"cur_op\" is not in the \"ops\" set. 
     """
     post_op = []
-    for idx, op in enumerate(ops):
-        if op == cur_op:
-            break
+    if search_all:
+        idx = -1
```

Comment:
Please add some comment explaining why you need to go through all ops instead of only looking after current op node.

Reply:
I added a comment here in the function description, 
https://github.com/PaddlePaddle/Paddle/blob/4cfeb39fa5ba2c7964f14efedac7ccad9ab3e118/python/paddle/fluid/contrib/mixed_precision/fp16_utils.py#L220
Do you think I should expand it?

Reply:
I rather thought about inplace comment explaining why you set `idx = -1` and in which situations you have to search in the way you do it right now. That would be useful note for future developers to understand the behavior. 
Actually the comment you wrote is misleading since even you use `search_all` you are still going through `ops` set, but from the beginning. I thought about noting why you do it like that, just for future.